Source/UI/Controllers/ThreedsWebViewController.swift

```diff
@@ -100,25 +143,43 @@ extension ThreedsWebViewController: WKNavigationDelegate {
         decisionHandler(dismissed ? .cancel : .allow)
     }
 
+    public func webView(_ webView: WKWebView, didFinish navigation: WKNavigation!) {
+        guard navigation == authUrlNavigation else {
+            return
+        }
+
+        logger?.log(.threeDSChallengeLoaded(success: true))
+    }
+
+    public func webView(_ webView: WKWebView, didFail navigation: WKNavigation!, withError error: Error) {
```

Comment:
can WKNavigation! be replaced by `WKNavigation` or  `WKNavigation?` and on line 146 also

Reply:
these are the methods from `WKNavigationDelegate`. We can't change the signature here